migrate_nidirect_node/migrate_nidirect_node_application/migrate_nidirect_node_application.module

```diff
@@ -13,20 +13,29 @@ use Drupal\migrate\Row;
  * Implements hook_migrate_MIGRATION_ID_prepare_row().
  */
 function migrate_nidirect_node_application_migrate_node_application_prepare_row(Row $row, MigrateSourceInterface $source, MigrationInterface $migration) {
-
   // Add a title to the body field.
   $body_field = $row->getSourceProperty('body');
 
-  if (!empty($body_field[0]['value'])) {
+  // Inject the title if the field has content and that content contains words.
+  // Checking the value has at least 1 vowel to prevent processing of
+  // &nbsp; etc.
+  if (!empty($body_field[0]['value']) && preg_match('/[aeiou]/m', $body_field[0]['value'])) {
```

Comment:
Clever idea ! Alternatively I suppose you could also have stripped the HTML entities first?

Reply:
We need to retain any html, this should be enough to filter out any fields that contain variations on <p>&nbsp;</p>